controllers/pods.go

```diff
@@ -36,16 +36,16 @@ func PodPortsWithXsite(i *infinispanv1.Infinispan) []corev1.ContainerPort {
 }
 
 func PodLivenessProbe() *corev1.Probe {
-	return probe(5, 10, 10, 1, 80)
+	return probe(5, 0, 10, 1, 80)
 }
 
 func PodReadinessProbe() *corev1.Probe {
-	return probe(5, 10, 10, 1, 80)
+	return probe(5, 0, 10, 1, 80)
 }
 
 func PodStartupProbe() *corev1.Probe {
-	// Maximum 10 minutes (60 * 10s) to finish startup
-	return probe(60, 10, 10, 1, 80)
+	// Maximum 10 minutes (600 * 1s) to finish startup
+	return probe(600, 10, 1, 1, 80)
```

Comment:
we discussed the initial delay to be 1 second too.

Reply:
Whoops, good catch!